### PR TITLE
Dev/jitsuineccles/sphinx documentation

### PR DESCRIPTION
**Pull request by jitsuineccles**

### CODE REVIEW

File: archivist/events.py [hunk 1]
```diff
@@ -69,15 +133,46 @@ def __query(props, attrs, asset_attrs):
...
+    def wait_for_confirmed(
+        self, *, asset_id=ASSETS_WILDCARD, props=None, attrs=None, asset_attrs=None
+    ):
+        """Wait for events to be confirmed.
+
+        Waits for all events that match criteria to be confirmed.
+
+        Args:
+            asset_id (str): optional asset identity e.g. assets/xxxxxxxxxxxxxxxxxxxxxxxxxx
+            props (dict): e.g. {"tracked": "TRACKED" }
```
Comment (JAG-UK) [inline archivist/events.py:165]: ‘Door’ is not a reasonable event display type, and doesn’t help to disambiguate event_attributes from asset_attributes. Leave the asset example as ‘door’ but change this event one to ‘open’

File: archivist/events.py [hunk 2]
```diff
@@ -107,7 +216,20 @@ def read_by_signature(
...
     ):
-        """docstring"""
+        """Read event by signature.
+
+        Reads event that meets criteria. Only one event is expected.
+
+        Args:
+            asset_id (str): optional asset identity e.g. assets/xxxxxxxxxxxxxxxxxxxxxxxxxx
+            props (dict): e.g. {"tracked": "TRACKED" }
+            attrs (dict): e.g. {"arc_display_type": "door" }
```
Comment (JAG-UK) [inline archivist/events.py:226]: Again event display_type should be ‘open’

File: archivist/locations.py [hunk 3]
```diff
@@ -1,28 +1,77 @@
...
+#: Default page size - number of entities fetched in one call to the
+#: :func:`~_LocationsClient.list` method.
 DEFAULT_PAGE_SIZE = 500
 
 
 class _LocationsClient:
-    """docstring"""
+    """LocationsClient
+
+    Access to locations entitiies using CRUD interface. This class is usually
```
Comment (JAG-UK) [inline archivist/locations.py:36]: Typo too many i’s

File: archivist/locations.py [hunk 4]
```diff
@@ -1,28 +1,77 @@
...
 
     def create(self, props, *, attrs=None):
-        """docstring"""
+        """Create location
+
+        Creates location with defined properties and attributes.
+
+        Args:
+            props (dict): properties for this location.
+            attrs (dict): attributes of created asset.
```
Comment (JAG-UK) [inline archivist/locations.py:55]: It’s a location not an asset

File: archivist/locations.py [hunk 5]
```diff
@@ -1,28 +1,77 @@
...
 
     def create_from_data(self, data):
-        """docstring
+        """Create location
+
+        Creates location with request body from data stream.
+        Suitable for reading data from a file using json.load or yaml.load
+
+        Args:
+            data (dict): list os accpted behaviours for this asset.
```
Comment (JAG-UK) [inline archivist/locations.py:70]: “Of accepted”

File: archivist/assets.py [hunk 6]
```diff
@@ -1,6 +1,24 @@
...
+   For example instantiate an Archivist instance and execute the methods of the class:
+
+   .. code-block:: python
+
+      with open(".auth_token", mode="r") as tokenfile:
+          authtoken = tokenfile.read().strip()
+
+      # Initialize connection to Archivist
+      arch = Archivist(
+          "https://soak-0-avid.engineering-k8s-stage-2.dev.wild.jitsuin.io",
```
Comment (JAG-UK) [inline archivist/assets.py:17]: Please change all references to soak into references to the rkvst.poc.jitsuin.io host

**Review by JAG-UK — CHANGES_REQUESTED** [review]

File: archivist/events.py (hunk 2 above)
Comment (jitsuineccles) [reply archivist/events.py]: fixed

File: archivist/locations.py (hunk 3 above)
Comment (jitsuineccles) [reply archivist/locations.py]: fixed

File: archivist/locations.py (hunk 4 above)
Comment (jitsuineccles) [reply archivist/locations.py]: fixed

File: archivist/locations.py (hunk 5 above)
Comment (jitsuineccles) [reply archivist/locations.py]: fixed

File: archivist/assets.py (hunk 6 above)
Comment (jitsuineccles) [reply archivist/assets.py]: fixed

**Review by JAG-UK — APPROVED** [review]

**Pull request merged**